contracts/library/LoanLib.sol

```diff
@@ -146,7 +146,7 @@ library LoanLib {
 
         // Check if the loan is past the defaultGracePeriod and that `msg.sender` has a percentage of total LoanFDTs that is greater
         // than the minimum equity needed (specified in globals)
-        return pastDefaultGracePeriod && balance >= totalSupply * _globals(superFactory).minLoanEquity() / 10_000;
+        return pastDefaultGracePeriod && balance >= (totalSupply * _globals(superFactory).minLoanEquity() / 10_000);
```

Comment:
If you wanted to be more explicit here:
```suggestion
        return pastDefaultGracePeriod && balance >= ((totalSupply * _globals(superFactory).minLoanEquity()) / 10_000);
```
or else we're arbitrarily worrying about `solc` handling of comparisons, but not BEDMAS.